src/bullets/__init__.py

```diff
@@ -1,6 +1,8 @@
-from .portfolio import *
-from .runner import *
-from .strategy import *
+from bullets.data_source import *
```

Comment:
This import is not necessary, if we have many data sources in the future, it's going to import all of them at once.

Reply:
should I import fmp_datasource or none at all?

Reply:
None